lib/horoshop.rb

```diff
@@ -1,7 +1,11 @@
 # frozen_string_literal: true
 
+require 'faraday'
+
 # Base class used to store data about authentication
 class Horoshop
+  ERROR = { server_error: { status: 'HTTP_ERROR', message: 'UNKNOWN SERVER ERROR' } }.freeze
+  EXPIRATION_TIME = Time.now + 600
```

Comment:
**expriation_time** This is not constant, after second request it will be changed. Don use constants instead variables.